CHANGES.txt

```diff
@@ -88,6 +88,11 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       for known exception types.
     - TEMPFILE: Fix the tempfile encoding test regex strings to eliminate
       warnings in python 3.12 and 3.13.
+    - Add tags to the remaining section headers in the User Guide. This is
```

Comment:
Looks like you put this in jcbrill's contributions? Is that an error?
Can you also add to RELEASE.txt?
Otherwise looks good to me.

Reply:
Yeah, good catch. updated.